### PR TITLE
Merge from Vishwa/promstandardmetrics into ci_feature

### PR DESCRIPTION
**Pull request by vishiy**
script changes for prometheus

### CODE REVIEW

File: Kubernetes/omsagent.yaml [hunk 1]
```diff
@@ -11,7 +11,9 @@ metadata:
 rules:
 - apiGroups: [""]
   resources: ["pods", "events", "nodes", "namespaces", "services"]
-  verbs: ["list"]
+  verbs: ["list", "get", "watch"]
+- nonResourceURLs: ["/metrics"]
```
Comment (r-dilip) [inline Kubernetes/omsagent.yaml]: Does this include a leading / or does it not matter? (the nonResourceURLs)

File: Kubernetes/omsagent.yaml [hunk 2]
```diff
@@ -398,7 +400,7 @@ spec:
          memory: 500Mi
         requests:
          cpu: 50m
-         memory: 100Mi
+         memory: 175Mi
```
Comment (r-dilip) [inline Kubernetes/omsagent.yaml]: revert to 100Mi?

File: Kubernetes/omsagent.yaml (hunk 1 above)
Comment (vishiy) [reply Kubernetes/omsagent.yaml]: its a URL. it is required to have /metrics.

File: Kubernetes/omsagent.yaml (hunk 2 above)
Comment (vishiy) [reply Kubernetes/omsagent.yaml]: i am still not sure if we should ship rs with fb & telegraf enabled for this release. But we definetely need to for next release. If we remove, i will revert. Also, leaving the requests raised, does not hurt much.

File: ci_feature/setup.sh [hunk 3]
```diff
@@ -14,7 +14,7 @@ wget https://github.com/Microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent
 #create file to disable omi service startup script
 touch /etc/.omi_disable_service_control
 
-wget https://github.com/Microsoft/Docker-Provider/releases/download/4.0.0-0/docker-cimprov-4.0.0-0.universal.x86_64.sh
+wget https://dockerprovider.blob.core.windows.net/prometheus/docker-cimprov-5.0.0-0.universal.x86_64.sh
```
Comment (r-dilip) [inline ci_feature/setup.sh]: Is the agent going to be released from this private storage account?

Comment (vishiy) [reply ci_feature/setup.sh]: this will change when we have a release build and move to ci_feature_prod. How else we test the build ? This is not new :)

**Pull request merged**